collectors/python.d.plugin/tor/tor.chart.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+# Description: adaptec_raid netdata python.d module
+# Author: Federico Ceratto <federico.ceratto@gmail.com>
+# Author: Ilya Mashchenko (l2isbad)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from bases.FrameworkServices.SimpleService import SimpleService
+
+try:
+    import stem
+    import stem.connection
+    import stem.control
+    STEM_AVAILABLE = True
+except ImportError:
+    STEM_AVAILABLE = False
+
+
+DEF_PORT = 'default'
+
+ORDER = [
+    'traffic',
+]
+
+CHARTS = {
+    'traffic': {
+        'options': [None, 'Tor Traffic', 'KB/s', 'traffic', 'tor.traffic', 'area'],
+        'lines': [
+            ['read', 'read', 'incremental', 1, 1024],
+            ['write', 'write', 'incremental', 1, -1024],
+        ]
+    }
+}
+
+
+class Service(SimpleService):
+    """Provide netdata service for Tor"""
+    def __init__(self, configuration=None, name=None):
+        SimpleService.__init__(self, configuration=configuration, name=name)
```

Comment:
Personally, I would prefer to see `super()` used here instead of explicitly calling out the parent class.  All of the framework services are new-style classes on Python 2.7 (they inherit from `SimpleService`, which inherits from `object`), so we can completely eliminate the explicit reference to the parent class with `super()`

Reply:
ok